release-notes.md

```diff
@@ -1,7 +1,47 @@
-# Release 2.1.17
+# Release 2.2.2
 
 ## What's New
-* none
+
+The automatic update process has changed! Prior to version 2.2.x, automatic upgrades were accomplished exclusively
+through the `ziti-monitor` service making a REST request to the GitHub API url. With 2.2.x this process will change.
```

Comment:
maybe worth adding/replacing so the takeaway is "whenever we published a release to the GH releases page..."